pkg/mimir/mimir_test.go

```diff
@@ -168,6 +169,9 @@ func TestMimir(t *testing.T) {
 		},
 		Frontend: frontend.CombinedFrontendConfig{
 			QueryEngine: "prometheus",
+			FrontendV2: v2.Config{
+				SchedulerAddress: "localhost",
+			},
```

Comment:
See [comment below](https://github.com/grafana/mimir/pull/12449/files#r2287010968) for explanation of this - previously this test passed due to the incorrect behaviour explained there.